Dockerfile

```diff
@@ -22,10 +22,20 @@ RUN apt-get update && \
         librabbitmq-dev \
         libyaml-cpp-dev \
         rapidjson-dev \
-        pybind11-dev \
+#        pybind11-dev \
         wget && \
     rm -rf /var/lib/apt/lists/*
 
+RUN cd /usr/local && \
+    git clone https://github.com/pybind/pybind11.git && \
+    cd pybind11 && \
+    git checkout master && \
```

Comment:
Should this instead be a semantic tag rather than grabbing master? (could make it an arg that defaults to our preferred SemVer so one could more easily test master or some other version w/o changing the Dockerfile)

Reply:
This makes sense.  I'll make the change, test, and merge.